### DISCUSSION TITLE
Allow player access osu!Forum in-game and do everything like on the website

### DISCUSSION THREAD

**Discussion by LIII-Works** [ppy/osu#13211] [Ideas] [answered]
### Describe the feature:
 Since we can visit update log webpage in osu!lazer, why don't add the ability for players to access osu!Forum in-game? Allow players do everything on webpage in-game. This can be a supporter feature to make supporter "worth more", generall player can still visit osu!Forum on browser. It is fair since the general player can still visit osu!Forum on browser

**Comment by peppy** [comment, marked as answer]
We do have the intention on adding every web view into the game eventually, but forums will definitely require a good amount of users want it to happen as the effort required to make it happen is considerable.
